crates/apps/src/lib/config/genesis/chain.rs

```diff
@@ -152,10 +152,21 @@ impl Finalized {
                 .map(|tx| Address::Established(tx.tx.data.address.raw.clone()))
                 .expect("Validator alias not found in genesis transactions.");
             wallet.extend_from_pre_genesis_validator(
-                address,
-                alias,
+                address.clone(),
+                alias.clone(),
                 validator_wallet,
-            )
+            );
+            let address_alias = alias::validator_address(&alias);
+            wallet.insert_address(
+                address_alias.normalize(),
+                address.clone(),
+                false,
+            );
```

Comment:
I think instead of storing the validator address this way, we could modify `extend_from_pre_genesis_validator` to store the validator address in the wallet.

something like this:
https://github.com/anoma/namada/blob/621cce94399e3ce5088e07dfff842f438b216103/crates/sdk/src/wallet/store.rs#L648-L649

Reply:
Done in [705ccd7](https://github.com/anoma/namada/pull/3093/commits/705ccd79423c74a25ff3450e91fe032bb79e0b9b)